.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -19,14 +19,26 @@ Optional breaking changes message. If your PR includes breaking changes. It is e
 
 ## Checklist
 
-<!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->
+- [ ] MDX documentation adheres to Canvas Kit's [standard MDX template](https://github.com/Workday/canvas-kit/discussions/1131)
```

Comment:
I added the Checklist back along with a single item referencing the MDX template. Let me know if you have any ideas on better places to reference it.

I'd like to move the MDX template out of a GH discussion and into something a bit more official at some point in the future.

Reply:
hey James,  just curious if there's a `COPY ALL OF THIS`. Me being lazy i know you've outlined it per section, do you think there's such a thing as a base template to copy and then modify per section?

Reply:
Yes! Love it.